### ISSUE TITLE
Fix Runner Log window Chinese and Latin encoding chars on Windows

### ISSUE THREAD

**Issue by HelioGuilherme66** [robotframework/RIDE#1983]
This is a split from #1892 where the major issue is the garbled output in Output window.
As it is now, that Output is OK, but not the Log window.

Actually, the output starts by being OK, but then other test makes it change. See here the text:
```
20190825 21:45:13.873 :  INFO : Slept 5 seconds
20190825 21:45:13.873 :  INFO : This is normal log: simplified Chinese: 汉语
Ending test:   Chinese çªºãâé.Chinese Tést.simplified Chinese: 汉语

Starting test: Chinese çªºãâé.Chinese Tést.Another € testº
20190825 21:45:13.873 :  INFO : ${options} = <selenium.webdriver.chrome.options.Options object at 0x0000027E2F48FDC8>
20190825 21:45:13.873 :  INFO : Creating an instance of the Chrome WebDriver.
20190825 21:45:17.369 :  INFO : &{mobile_emulation} = { deviceName=Galaxy S5 }
20190825 21:45:17.369 :  INFO : Creating an instance of the Chrome WebDriver.
20190825 21:45:19.953 :  INFO : Opening url 'https://www.google.com'
Ending test:   Chinese 窺䣩.Chinese T괴.Another  testڀ
```
We can see the test name is correctly logged at start (`Starting test: Chinese çªºãâé.Chinese Tést.Another € testº`, but then at end the characters are modified and originally shown as hexadecimal.

This is really a corner case of mixed encodings, and as it shows badly on the Log window but not on the Log.html, we leave this for someday :).

**Comment by spitamatus** [comment]
Hello. I am running

Windows 7
Python 3.6.8
RIDE 1.7.4.2

and the problem still persists for me. The output in both output windows is showing latin chars incorrectly . The possible fixes I found as solution are:

1. #2053 
2. in #2170 as mentioned by @gogily

Are you planning on fixing this?

**Comment by HelioGuilherme66** [comment]
> The output in both output windows is showing latin chars incorrectly

@spitamatus Please help us to understand what would be the difference on your system.
Please create and attach a small runnable test suite that we can run on our systems (I still have a Windows 7 around ;) ).
Run it and capture a screenshot of RIDE with both windows.
Paste here, the output of (from a command window):
`chcp`
`python --version`
`python -m pip list`
`echo %PYTHONIOENCODING%`

**Comment by spitamatus** [comment]
Thank you for quick response. I am attaching a zip which consists of a test suite as .robot file and output from the command line in the form of a .txt file. 
[Test.zip](https://github.com/robotframework/RIDE/files/4825717/Test.zip)

I have no idea why `echo %PYTHONIOENCODING%` has the value of %PYTHONIOENCODING%, but when I run the code:
```
import sys

if __name__ == "__main__":
	print(sys.getdefaultencoding())
	print(sys.getfilesystemencoding())
```
I get the output:
```
utf-8
utf-8
```
And finally, the screenshot of RIDE:

![RIDE_screen](https://user-images.githubusercontent.com/48322507/85568878-5c37c400-b632-11ea-9c64-6a6b6385f929.PNG)

**Comment by HelioGuilherme66** [comment]
@spitamatus You can now test the current development from `master` on python 3. You must have `set PYTHONIOENCODING=utf-8` for it to work (or add it to system variables).
`pip install -U https://github.com/robotframework/RIDE/archive/master.zip`

**Comment by HelioGuilherme66** [comment]
Closing because can be tested on soon to be pre-released, v2.0b1.
